### DISCUSSION TITLE
Automatically reset resource-world

### DISCUSSION THREAD

**Discussion by sakurawald** [Zrips/CMI#7246] [Feature Requests]
Automatically reset some resource-world (overworld, nether, the_end), this function is very useful for some survival server,
you can set the resource-world names and then the plugin auto reset (delete and create the worlds), set some gamerules (like keepInventory), set the worldborder.
Here is a open-source plugin do this thing, but the author is busy: https://www.spigotmc.org/resources/%E2%9C%85-resource-world-%E2%9C%85-never-run-out-of-resources-must-have.75994/
This plugin has some bugs about the resouse-world portals.

I know the CMI has a powerful schedule, but some commands have to use in-game, can't performed by the console. like: /worldborder, /gamerule, and you can't use vanilla command to create/delete a world.
if there is some support from a plugin ,you'll do this things easier.

**Comment by mrfloris** [comment]
there's asFakeOp! to run as operateor ingame, might be handy for those commands that don't have console support. 
On the minigame server I use cmi scheduler.yml to help reset a few areans daily. 
I make a custom alias command with cmi aliaseditor, like `/_worldmanager` where I read the params from and parse them $1 $2 $3 etc as I need them. negated the perm for this alias for the default group. And then use scheduler.yml to point to that command, feeding it the params I need for that schedule.

**Comment by sakurawald** [comment]
So helpful of you, thank you.
i will read the asFaskOp! documention.

**Comment by sakurawald** [comment]
asFakeOp! doesn't support which world the fake player is.
some commands use a real player's current location to perform some logis.
like /worldborder and /gamerule.
so asFakeOp! doesn't support this.
